### ISSUE TITLE
Foldername does not get proper Title name after "Move file to new folder"

### ISSUE THREAD

**Issue by shad00m** [DoctorD1501/JAVMovieScraper#102]
v0.2.04-alpha

happened to:
agemix176b
air040
atfb153

I scraped with "Scrape Adult Amalgamated JAV Censored Group"
checked all the info to make sure its correct
clicked "Move File to New Folder"

Expected:
It will create a folder according to the title (e.g. agemix176b title is Tantalizing Tight Skirt - Ass Humping) so it would be c:\educational_movies\Tantalizing Tight Skirt - Ass Humping\

What it actually did:
it created a folder called the same as the filename, c:\educational_movies\agemix176b

I repeated the scraping process multiple times but it always created a folder not fitting to the title

**Comment by shad00m** [comment]
I noticed this stacktrace on the movie that does not have the proper name:
http://hastebin.com/hanipecobi.log

**Comment by DoctorD1501** [comment]
It might make sense to look at the rename settings and make sure that's enabled in preference menu instead of using the "Movie File To New Folder" feature. You can customize the name that it will make in the rename settings menu. When you write the movie after it's been scraped, it will rename it to the correct folder. You can even use the rename feature on already scraped movies by just clicking on the movie in the list and clicking write file data again.

The stack trace you pasted with the date was actually pretty interesting. I'm not on german windows and I'm not getting that error when scraping the exact same movie, but I added a "Locale.English" to all instances where I create a Date object to see if that fixes things. Let me know if you still get an error from scraping from r18.

You can download a new dev build that has the fix for the german localization problem (hopefully!) from here:

http://www.mediafire.com/download/pm3d2yl49qa99fe/JAVMovieScraper.jar

Thanks and let me know how it goes,
DoctorD

**Comment by shad00m** [comment]
The stacktrace is gone with the build you posted, thank you.
About the renaming, I'm not sure if it really has something to do with the renaming settings as it works with other movies with the exact same settings.

I've just noticed that it will fail creating the correct folder name when there is no "OriginalTitle"
some examples:
APAM-004 and AGEMIX-164 have an original title in japanese, and after moving them, it created the proper english folder.
AGEMIX-176, AIR-040 and ATFB-153 have NO original title in japanese, after moving them it created a folder same as the filename with the same rename settings. Even though the rename settings window clearly shows me proper example output

here the screens:
Doesnt work: http://i.imgur.com/kjWK6C4.jpg
Works: http://i.imgur.com/CNK4g9U.jpg

**Comment by DoctorD1501** [comment]
I'm not sure why you're getting no original title. Maybe try deleting your amalgamation settings json file (in the same directory as the program) and restart the program. I'm seeing the program scrape an original title for all the movies you say are not scraping with one. I suppose it's possible it could be some side effect of us being in different regions.

When you scrape AIR-040, what are the results you see in the console log?

That said, I think the move to new folder function needs a complete rewrite. I wrote it before there was any kind of renaming available in the program, and I just wanted something quick and dirty to suit my needs. Later on, a contributor added in the first version of the rename system and I've been improving that over time. The actual rename screen's settings do not have any effect on the movie to new folder function but instead only when you hit the "Write File Data Button".

Now that the renamer screen supports allowing you to move files into new folders, the "Move File to Folder" button is completely obsolete and you should be using the "Write File Data" function instead to move files around, except in the case where you have an unscraped movie file (like mp4, avi, etc) and you just want to make a quick folder for it with the same name as the file, more or less.

**TLDR**: I'm going to make it a priority as one of my next projects to try to make the "Move File to New Folder" button respect the options the user has chosen in the rename settings dialog.

**Comment by shad00m** [comment]
While fiddling around what caused it not finding the original title. I had to clean the file name first.
With filename air040-asiamoviepass.mp4 it wasnt able to find the original title but everything else.
With filename air040.mp4 it was able to find the original title and everything else, and even moving to folder worked with the correct folder name(!!).

Output when scraping air040-asiamoviepass.mp4:
http://hastebin.com/axeteyicad.log (folder name is just filename)
Output when scraping air040.mp4:
http://hastebin.com/tigaqeforu.log (folder created successfully)

With the cleaned filename it was able to scrape from other websites instead of only r18.com where it got the originaltitle from as r18.com doesnt seem to have any.
So I guess having no originaltitle is connected to not creating the folder with the proper foldername ?

**Comment by DoctorD1501** [comment]
This should be fixed now in the newest version. It will use rename settings instead.
